### PR TITLE
[CS] Code Style fixes for administrator/components/com_finder/

### PR DESCRIPTION
**Pull request by photodude**
Pull Request for Issue code style in com_finder

### Summary of Changes
- Multi-line function call not indented correctly;
- Line indented incorrectly
- Logical operator "or" not allowed; use "||" instead
- Logical operator "and" not allowed; use "&&" instead
- Please consider an empty line before the if statement;

[Automatically fixed with Joomla code standards 2.0.0 PHPCS2-alpha2 fixers](https://github.com/joomla/coding-standards/releases/tag/2.0.0-alpha2)

None of the manual only fixes have been applied

### Testing Instructions
Merge by code review

### Expected result
code style has been applied as listed above, old code style on drone does not error.

### Actual result
code style had not been applied. [Autofixers from the Joomla code standards 2.0.0 PHPCS2 alpha2](https://github.com/joomla/coding-standards/releases/tag/2.0.0-alpha2) were used to impliment fixable codestyle

### Documentation Changes Required
none

### CODE REVIEW

File: administrator/components/com_finder/helpers/indexer/stemmer/porter_en.php [hunk 1]
```diff
@@ -261,7 +261,7 @@ private static function step4($word)
 				break;
 			case 'c':
 					self::replace($word, 'ance', '', 1)
```
Comment (Quy) [inline administrator/components/com_finder/helpers/indexer/stemmer/porter_en.php]: Remove tab.

File: administrator/components/com_finder/helpers/indexer/stemmer/porter_en.php [hunk 2]
```diff
@@ -294,7 +294,7 @@ private static function step4($word)
 				break;
 			case 't':
 					self::replace($word, 'ate', '', 1)
```
Comment (Quy) [inline administrator/components/com_finder/helpers/indexer/stemmer/porter_en.php]: Remove tab.

File: administrator/components/com_finder/helpers/indexer/stemmer/porter_en.php (hunk 1 above)
Comment (photodude) [reply administrator/components/com_finder/helpers/indexer/stemmer/porter_en.php]: That is out of the scope of this PR as it's in the existing file and was not modified by the Joomla coding standard PHPCS2 auto fixers

A separate PR should be opened to remove the specified tab

File: administrator/components/com_finder/helpers/indexer/stemmer/porter_en.php (hunk 2 above)
Comment (photodude) [reply administrator/components/com_finder/helpers/indexer/stemmer/porter_en.php]: That is out of the scope of this PR as it's in the existing file and was not modified by the Joomla coding standard PHPCS2 auto fixers

A separate PR should be opened to remove the specified tab

File: administrator/components/com_finder/helpers/indexer/stemmer/porter_en.php (hunk 1 above)
Comment (zero-24) [reply administrator/components/com_finder/helpers/indexer/stemmer/porter_en.php]: done https://github.com/joomla/joomla-cms/pull/16986

File: administrator/components/com_finder/helpers/indexer/stemmer/porter_en.php (hunk 2 above)
Comment (zero-24) [reply administrator/components/com_finder/helpers/indexer/stemmer/porter_en.php]: done https://github.com/joomla/joomla-cms/pull/16986

File: administrator/components/com_finder/helpers/indexer/driver/postgresql.php [hunk 3]
```diff
@@ -320,7 +320,8 @@ public function index($item, $format = 'html')
 		 * table have a term of 0, then no term record exists for that
 		 * term so we need to add it to the terms table.
 		 */
-		/* Emulation of IGNORE INTO behaviour */
+		// Emulation of IGNORE INTO behaviour
+
```
Comment (zero-24) [inline administrator/components/com_finder/helpers/indexer/driver/postgresql.php:324]: I'm not sure about this clean line? If I understand this correct the coment is about the folowing lines of code?

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 94359ebcb5d5e5ec13dbd788f754bb728ea38059<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16915">issues.joomla.org/tracker/joomla-cms/16915</a>.</sub>

File: administrator/components/com_finder/helpers/indexer/driver/postgresql.php (hunk 3 above)
Comment (photodude) [reply administrator/components/com_finder/helpers/indexer/driver/postgresql.php:324]: I would agree that we need to add a line before this comment to separate from the other comment. I would suggest that change is also beyond the scope of this PR as that spacing would be a manual change and not something automatically changed by the auto fixers.

**Review by dgrammatiko — APPROVED** [review]

**Comment by photodude** [comment]
@mbabker @wilsonge bump for merge consideration

**Comment by zero-24** [comment]
Merged Thanks 👍 I'm going to take a look into the comment thing now

**Comment by photodude** [comment]
Thanks @zero-24

**Pull request merged**